src/components/Header/index.tsx

```diff
@@ -8,10 +8,19 @@ import DropdownMenu from '@components/DropdownMenu/DropdownMenu';
 
 const Header: React.FC = () => {
     const [openMenu, setOpenMenu] = useState(false);
+    const isAdmin = localStorage.getItem('admin');
```

Comment:
Maybe you should consider making one localStorage item, something like loginType (admin or user) and then check cases by value of loginType.

Reply:
@SreckoStojic out of scope for this task, we will handle this fix once BE for authentication is done